yelp_package/Makefile

```diff
@@ -54,7 +54,7 @@ go_entrypoint:
 	rm -rf gopath && \
 	mkdir -p gopath && \
 	cd gopath && \
-	wget https://github.com/Yelp/paasta-tools-go/archive/v$(PTG_VERSION).zip && \
+	wget https://github.com/Yelp/paasta-tools-go/archive/v$(PTG_VERSION).zip -O v$(PTG_VERSION).zip && \
```

Comment:
do we need to set the filename? from reading the manpage, it seems like we should be able to just use -O (or preferably the --long-form of this option `--remote-name`) to ensure that the file is named the same as it was on the remote side

```
-O, --remote-name
              Write output to a local file named like the remote file we get. (Only the file part of the remote file is used, the path is cut off.
```